tests/integration/test_vulnerability_detector/test_providers/test_update_from_year.py

```diff
@@ -37,6 +37,12 @@
     - Red Hat 8
     - Ubuntu Focal
     - Ubuntu Bionic
+    - SUSE Linux Enterprise Desktop 11
+    - SUSE Linux Enterprise Desktop 12
+    - SUSE Linux Enterprise Desktop 15
+    - SUSE Linux Enterprise Server 11
+    - SUSE Linux Enterprise Server 12
+    - SUSE Linux Enterprise Server 15
```

Comment:
In this case, we only test SUSE 11 desktop. It should remove the others.

```suggestion
    - SUSE Linux Enterprise Desktop 11
```